src/mongo/installer/msi/wxs/UIFragment.wxs

```diff
@@ -2,11 +2,44 @@
 <Wix xmlns="http://schemas.microsoft.com/wix/2006/wi"  xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
     <Fragment>
         <UI Id="MongoWixUI">
-            <!-- Base our MSI on the FeatureTree based Wix set, and add our custom dialog boxes -->
-            <UIRef Id="WixUI_FeatureTree" />
+            <!-- Based on WixUI_FeatureTree, with our custom dialog boxes -->
+          <UIRef Id="WixUI_Common" />
+          <TextStyle Id="WixUI_Font_Normal" FaceName="Tahoma" Size="8" />
+          <TextStyle Id="WixUI_Font_Bigger" FaceName="Tahoma" Size="12" />
+          <TextStyle Id="WixUI_Font_Title" FaceName="Tahoma" Size="9" Bold="yes" />
 
-            <Publish Event="NewDialog" Value="MongoSetupTypeDlg" Dialog="LicenseAgreementDlg" Control="Next">LicenseAccepted = "1"</Publish>
-            <Publish Event="NewDialog" Value="MongoSetupTypeDlg" Dialog="CustomizeDlg" Control="Back">WixUI_InstallMode = "InstallCustom"</Publish>
+          <Property Id="DefaultUIFont" Value="WixUI_Font_Normal" />
+          <Property Id="WixUI_Mode" Value="FeatureTree" />

```

Comment:
Ah, I think I see now. Is simply the more modern way of opting into `FeatureTree`?